python/tests/test_nessie_cli.py

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env python
-# -*- coding: utf-8 -*-
 #
```

Comment:
nit: still 2 lines before copy right... shouldnt the check be catching that now?

Reply:
The test ensures there is a commented empty line between shebang and license. Should I change this?

Reply:
in all the other files you have removed this empty line... only in 3 lines you have kept it.

which made me wonder if the regexp is working as intended...

generally both would be fine with me but if we remove the `coding` line we should do it uniformally across all the files imo

Reply:
These 3 files have the shebang while the others didn't. The regex expects that either no comment exists before license header or a shebang followed by a commented empty line.

Reply:
ah i see, sorry i missed that earlier.

but taking a step back:
none of these files need to be executable directly from a shell (there is nothing special about them compared to the other py files)... so we should just remove the shebang here, wdyt?

Reply:
removed shebangs from files and regex